### ISSUE TITLE
Windows system interruptions

### ISSUE THREAD

**Issue by goncalvesjean** [FrescoLogic/FL2000#40]
I'm using an HDMI adaptor to connect my third monitor when Windows install Fresco Logic FL2000 USB Display Adapter starts the systems interruptions as showing the attached image. When the interruption occurs, the monitor starts to blink...

![image](https://user-images.githubusercontent.com/32856006/75870126-4f743380-5de9-11ea-9d2c-741f94501047.png)

**Comment by fresco-fl2000** [comment]
yes, it is a FL2000 vulnerability. The design of FL2000 is that it requires CPU to keep driving traffic without a break. This means the driver uses a lot of "DPC" time to maintain the flow. There is nothing we can do about it. It is just what it is.
